README.md

```diff
@@ -1,2 +1,49 @@
+[![Build Status](https://travis-ci.org/ucsc-cgp/bouncer.svg?branch=master)](https://travis-ci.org/ucsc-cgp/bouncer)
+
 # bouncer
 The whitelist checker for authentication with CGP HCA Data Store
+
+## setup
+
+### regular
+Either add `cgp-bouncer` to your project requirements or
+```
+pip install cgp-bouncer
+```
+in a Python 3 virtualenv.
+
+### for development
+Instead of the steps above clone the repo, `cd` into the repo, and run
+```
+pip install -e .
+```
+
+To test, run
+```
+python -m unittest -v test.py
+```
+## how to use
+Using is simple!
+
+Here's an example
+
+```python
+>>> from bouncer import Bouncer
+>>> b = Bouncer('commons/dev/whitelist')
+>>> b.is_authorized('jrbrenna@ucsc.edu')
+True
+>>> b.is_authorized('evil.gnomes@ucsc.edu')
+False
+```
+
+This checks the AWS Secret Keeper called `commons/dev/whitelist` to see
+if the users `jrbrenna@ucsc.edu` and `evil.gnomes@ucsc.edu` are in the
+whitelist.
+
```

Comment:
Please add instructions for the initial setup of a whitelist in Secrets Manager.
It is very much like the "adding users to the whiltelist" section below, with a few more steps, such as specifying the key name "email" and the secret name. I propose copying the "adding users to the whiltelist" section below and adding/modifying steps as appropriate.

Reply:
Good point. Adjusting the docs...